### PR TITLE
User module

### PR DESCRIPTION
**Pull request by mberestovska**

### CODE REVIEW

File: src/app/communication/models/id.object.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export interface IIdObject {
     id: number;
+    ID?: number;
```
Comment (MykolaAndrushchack) [inline src/app/communication/models/id.object.ts]: why this is two id?

File: src/app/user/user-details/user-details.component.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@import '@angular/material/prebuilt-themes/deeppurple-amber.css';
```
Comment (MykolaAndrushchack) [inline src/app/user/user-details/user-details.component.scss]: we can import this theme in angular.json

File: src/app/user/user-edit/user-edit.component.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@import '@angular/material/prebuilt-themes/deeppurple-amber.css';
```
Comment (MykolaAndrushchack) [inline src/app/user/user-edit/user-edit.component.scss]: repeating import

File: src/app/communication/communication.module.ts [hunk 4]
```diff
@@ -15,7 +15,11 @@ export class CommunicationModule {
                     provide: CommunicationConfig,
                     useExisting: communicationConfigToken,
                 },
-                registerService(UsersProvider, FakeUsersProvider, HttpUsersProvider),
+                registerService(UsersProvider, FakeUsersProvider, HttpUsersProvider)
+                // {
```
Comment (ravlik) [inline src/app/communication/communication.module.ts]: remove comments

File: src/app/communication/services/http/http.provider.ts [hunk 5]
```diff
@@ -63,7 +63,10 @@ export abstract class HttpProvider<T extends IIdObject> implements Provider<T> {
     // }
 
     private _concatUrl(...params: (string | number)[]): string {
-        return `${this._baseUrl}`.concat(...params.map(toString));
+        console.log(`${this._baseUrl}`);
```
Comment (ravlik) [inline src/app/communication/services/http/http.provider.ts]: remove this

**Pull request merged**